core/store/orm/config.go

```diff
@@ -508,6 +555,36 @@ func (c Config) OCRContractTransmitterTransmitTimeout() time.Duration {
 	return c.viper.GetDuration(EnvVarName("OCRContractTransmitterTransmitTimeout"))
 }
 
+func (c Config) getDurationWithOverride(override time.Duration, field string) time.Duration {
+	if override != time.Duration(0) {
+		return override
+	}
+	return c.viper.GetDuration(EnvVarName(field))
+}
+
+func (c Config) OCRObservationTimeout(override time.Duration) time.Duration {
```

Comment:
Perhaps the override could be optional?

Reply:
it is optional, just passing an empty time.Duration will fallback to global. I didn't want to make it a pointer because they you lose the one line capability and have to make a bunch of tmp vars to use the api which is annoying

Reply:
I was imagining like this:

`(override ...time.Duration)` 

But I'm OK with either way